src/minicraft/entity/furniture/Spawner.java

```diff
@@ -176,7 +171,7 @@ public boolean interact(Player player, Item item, Direction attackDir) {
 			
 			return true;
 		}
-		
+
```

Comment:
if there's one thing I hate it's having the whitespace on blank lines removed... then again that's a personal preference, and since I don't work on this anymore and it's not exactly consistent throughout anyway I won't bother with it.